tests/feed/test_feed.py

```diff
@@ -77,3 +77,29 @@ def test_posts_order_in_dict_by_creation_date(self, num_of_posts, persist_user,
 
         posts_for_user = list(PostManager.get_posts_for_user(persist_user).keys())
         assert all(earlier.date <= later.date for earlier, later in zip(posts_for_user, posts_for_user[1:]))
+
+
+@pytest.mark.django_db
+class TestPostsListView:
+    def test_posts_list_view_loaded_authorized(self, authorized_client):
+        post_list_response = authorized_client.get('/feed/list/')
+        assert post_list_response.status_code == 200
+
+    def test_Posts_list_view_loaded_unauthorized(self, client):
```

Comment:
test name should include only lower letters

Reply:
done